src/client/helpers/entity.tsx

```diff
@@ -256,6 +256,7 @@ export function getEntityUrl(entity) {
 export const ENTITY_TYPE_ICONS = {
 	Area: faGlobe,
 	Author: faUser,
+	Book: faBookOpen,
```

Comment:
I'm still wondering if we should use the classic "wizard" icon of a magic wand.
I'm not a fan of the font awesome one… https://fontawesome.com/icons/wand-magic-sparkles?s=solid